src/psbt.cpp

```diff
@@ -496,6 +496,12 @@ bool PSBTOutput::Merge(const PSBTOutput& output)
     return true;
 }
 
+CTxOut PSBTOutput::GetTxOut() const
+{
+    assert(amount != nullopt);
+    return CTxOut(*amount, script);
```

Comment:
`*script` (`script` is an optional too)

Reply:
script is a `CScript`, not an optional. Should it be?

Reply:
I get a compile error about this when it is introduced, but it is fixed several commits later (in the commit that adds `nullopt` initializers to all the options). It may be that I am misinterpreting the compiler error and that it's somehow about initialization rather than about types, but regardless, this commit doesn't build.

Reply:
Specifically a compiler error about not finding an instance of `CTxOut()` which works for the given parameter types. I forget the exact error.

Reply:
Oh, script is a CScript originally, then made into an optional in a later commit.